Applications/Terminal/Terminal.cpp

```diff
@@ -265,7 +320,7 @@ void Terminal::escape$H(const ParamVector& params)
         row = params[0];
     if (params.size() >= 2)
         col = params[1];
-    set_cursor(row - 1, col - 1);
+    set_cursor(row - 1 + m_scroll_region_top, col - 1);
```

Comment:
This should only be done when origin mode is set. It looks like you started adding parsing for this (h/l), but didn't finish?